includes/Modules/Ads/Enhanced_Conversions.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * Class Google\Site_Kit\Modules\Ads\Enhanced_Conversions
+ *
+ * @package   Google\Site_Kit\Modules\Ads
+ * @copyright 2025 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Modules\Ads;
+
+use Google\Site_Kit\Context;
+
+/**
+ * Class Enhanced_Conversions.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class Enhanced_Conversions {
+
+	/**
+	 * Context instance.
+	 *
+	 * @since 1.128.0
+	 * @var Context
+	 */
+	private $context;
```

Comment:
I realise this was included in the IB; however, now that I think about it further, I don't think it will really be needed. We shouldn't add it unless explicitly required, so let's remove it.

```suggestion
```